test_util/test_aws_vpc.py

```diff
@@ -191,20 +159,41 @@ def main():
     vpc = None  # Set if the test owns the VPC
 
     if options.host_list is None:
-        log.info('CCM_VPC_HOSTS not provided, requesting new VPC from CCM...')
-        vpc = make_vpc(use_bare_os=options.test_install_prereqs)
-        host_list = vpc.hosts()
+        log.info('VPC_HOSTS not provided, requesting new VPC ...')
+        random_identifier = ''.join(random.choice(string.ascii_uppercase + string.digits) for _ in range(10))
```

Comment:
nit: We have this "make a random string of length X then append to prefix" a few places, might want to pull it into a common util.